build.gradle

```diff
@@ -38,12 +38,20 @@ repositories {
 dependencies {
     compile 'org.slf4j:slf4j-api:1.7.12'
 
-    def tapestry_version='5.3.7'
+    def tapestry_version='5.5.0-alpha-1'
```

Comment:
No, obviously not needed to depend on 5.5 but that's where I'm going to merge it in. I'm planning on merging the code directly into tapestry-jpa module for the 5.5 branch (currently in master).